common/frame_session.go

```diff
@@ -1029,7 +1023,11 @@ func (fs *FrameSession) attachIFrameToTarget(ti *target.Info, sid target.Session
 		return nil
 	}
 	// Remove all children of the previously attached frame.
-	fs.manager.removeChildFramesRecursively(fr)
+	err := fs.manager.removeChildFramesRecursively(fr)
+	if err != nil {
+		return fmt.Errorf("removing childs from frame with ID %v : %w",
```

Comment:
Instead of returning an error, can we log an error and continue? That way we came the same behaviour as before.

Reply:
done